docs/content/media/webdav.md

```diff
@@ -0,0 +1,46 @@
+# WebDAV
+
+You can use any [WebDAV](https://en.wikipedia.org/wiki/WebDAV) server to handle your image uploads in HedgeDoc.
+
+The WebDAV server must host the files in a way that allows HedgeDoc to request and receive them.
+
+You just add the following lines to your configuration:  
+<small>(with the appropriate substitution for `<CONNECTION_STRING>`, `<UPLOAD_DIR>`, and `<PUBLIC_URL>` of course)</small>
+```
+HD_MEDIA_BACKEND="webdav"
+HD_MEDIA_BACKEND_WEBDAV_CONNECTION_STRING="<CONNECTION_STRING>"
+HD_MEDIA_BACKEND_WEBDAV_UPLOAD_DIR="<UPLOAD_DIR>"
+HD_MEDIA_BACKEND_WEBDAV_PUBLIC_URL="<PUBLIC_URL>"
+```
+
+The `<CONNECTION_STRING>` should include the username and password (if needed) in the familiar way of `schema://user:password@url`.
+
+With `<UPLOAD_DIR>` you can specify a folder you want to upload to, but you can also omit this, if you prefer to upload directly to the root of the WebDAV server.
```

Comment:
Omitting means setting `HD_MEDIA_BACKEND_WEBDAV_UPLOAD_DIR` so an empty string of not setting it?

Reply:
clarified